github/prci.py

```diff
@@ -48,6 +48,7 @@ def __call__(self, depends_results={}):
         try:
             url = subprocess.check_output(cmd, shell=True)
         except subprocess.CalledProcessError as err:
+            logging.error(traceback.print_exc())
```

Comment:
Is there a reason to use traceback module over `logging.error(err, exc_info=True)` ?

Reply:
I don't think so. Doing some tests and searching for it, it seems that there is no difference. We can shift to use `logging.error` in order to have just one way of doing it.